ui/app/AppMain.qml

```diff
@@ -171,7 +171,7 @@ StatusAppLayout {
                 }
 
                 StatusMenuItem {
-                    enabled: false//chatsModel.communities.observedCommunity.admin
+                    enabled: chatsModel.communities.observedCommunity.admin
```

Comment:
Re-enabled this. Has been disabled for quite a while but I doubt that this is desired at the moment.